libde265/de265.h

```diff
@@ -378,13 +378,14 @@ enum de265_param {
   DE265_DECODER_PARAM_DUMP_VPS_HEADERS=2,
   DE265_DECODER_PARAM_DUMP_PPS_HEADERS=3,
   DE265_DECODER_PARAM_DUMP_SLICE_HEADERS=4,
-  DE265_DECODER_PARAM_ACCELERATION_CODE=5,   // (int)  enum de265_acceleration, default: AUTO
-  DE265_DECODER_PARAM_SUPPRESS_FAULTY_PICTURES=6, // (bool)  do not output frames with decoding errors, default: no (output all images)
-
-  DE265_DECODER_PARAM_DISABLE_DEBLOCKING=7,   // (bool)  disable deblocking
-  DE265_DECODER_PARAM_DISABLE_SAO=8           // (bool)  disable SAO filter
-  //DE265_DECODER_PARAM_DISABLE_MC_RESIDUAL_IDCT=9,     // (bool)  disable decoding of IDCT residuals in MC blocks
-  //DE265_DECODER_PARAM_DISABLE_INTRA_RESIDUAL_IDCT=10  // (bool)  disable decoding of IDCT residuals in MC blocks
+  DE265_DECODER_PARAM_DUMP_IMAGE_DATA=5,
```

Comment:
The values of existing params should not change to keep compatibility with existing applications. Just add the new property at the end and assign a previously unused value.

Reply:
done